vhdbuilder/packer/install-dependencies.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
+
+OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
+UBUNTU_OS_NAME="UBUNTU"
+MARINER_OS_NAME="MARINER"
+
 source /home/packer/provision_installs.sh
+source /home/packer/provision_installs_distro.sh
```

Comment:
Not your fault - I noticed there is go template syntax in the install file, not sure if this has negative impact to the vhd build, but I think it's regression from the recent containerd work. We should either move those go template syntax out of the install file (move to cse_config.sh), or move those static methods from cse_install.sh to cse_helpers so we don't need to source cse_install.sh here. 
https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/9444055

FYI @eriksywu